portfolio/src/main/webapp/infiniterun/gamestyle.css

```diff
@@ -0,0 +1,74 @@
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+}
+
+a.button {
+  background-color: #008CBA; /* Blue */
+  border: none;
+  color: white;
+  padding: 16px 32px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 4px 2px;
+  transition-duration: 0.4s;
+  cursor: pointer;
+}
+
+a.buttonblue {
+  background-color: white; 
```

Comment:
Seems like you can put this `background-color` in the parent style too? It's always white?

Reply:
It changes when you hover so that is why I kept it in the child style